recipes/rdkit/bld.bat

```diff
@@ -0,0 +1,40 @@
+rem surely there's a better way than this
+if "%PY_VER%"=="2.7" (
+	set PYTHON_LIBRARY=python27.lib
+) else if  "%PY_VER%"=="3.4" (
+	set PYTHON_LIBRARY=python34.lib
+) else if  "%PY_VER%"=="3.5" (
+	set PYTHON_LIBRARY=python35.lib
+) else if  "%PY_VER%"=="3.6" (
+	set PYTHON_LIBRARY=python36.lib
+) else (
+	echo "Unexpected version of python"
+	exit 1
+)
+
+cmake ^
+    -G "NMake Makefiles JOM" ^
+    -D RDK_INSTALL_INTREE=OFF ^
+    -D RDK_BUILD_INCHI_SUPPORT=ON ^
+    -D RDK_BUILD_AVALON_SUPPORT=ON ^
+    -D RDK_USE_FLEXBISON=OFF ^
+    -D Python_ADDITIONAL_VERSIONS=${PY_VER} ^
+    -D PYTHON_EXECUTABLE="%PYTHON%" ^
+    -D PYTHON_INCLUDE_DIR:PATH="%PREFIX%\include" ^
+    -D PYTHON_LIBRARY="%PREFIX%\libs\%PYTHON_LIBRARY%" ^
+    -D PYTHON_INSTDIR:PATH="%SP_DIR%" ^
+    -D BOOST_ROOT:PATH="%LIBRARY_PREFIX%" -D Boost_NO_SYSTEM_PATHS=ON ^
+    -D CMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+    -D CMAKE_BUILD_TYPE=Release ^
+    .
```

Comment:
Sorry, missed this before. We need the following after every command in the batch file. Comments and the `if` statements above can be ignored.

```batch
if errorlevel 1 exit 1
```